content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
+1. Install KKP master on one cluster (c1) and also use this cluster as seed (master-seed combo cluster)
+1. Make 2nd cluster (c2) as dedicated seed
+1. ArgoCD will be installed in each of the seed (and master/seed) to manage the respective seed's KKP components
+
+A high-level procedure to get ArgoCD to manage the seed would be as follows:
+1. Setup a kubernetes cluster to be used as master (or master-seed combo)
+1. Install ArgoCD helm chart and KKP ArgoCD Applications helm chart
+1. Install KKP on the master seed using kubermatic-installer
+1. Setup DNS records and Git repository tag so that ArgoCD can sync
+1. Sync the applications
+1. Repeat the procedure for 2nd seed (except installation of KKP on the seed)
+1. Add kkp seed record in master seed config files so that master takes control of seed cluster.
+1. Commit, push and re-tag the above change so that you can add the seed to master via ArgoCD UI
+
+Folder and File Structure section in the [README.md of ArgoCD Apps Component](https://github.com/kubermatic/kubermatic/blob/main/charts/gitops/kkp-argocd-apps/README.md) explains what files should be present for each seed in what folders and how to customize behavior of ArgoCD apps installation.
+
+**Note:** Configuring values for all the components of KKP is a humongous task. Also - each customer might like a different directory structure to manage KKP installation. This ArgoCD Apps based approach is an _opinionated attempt_ to provide a standard structure that can be used in most of the customer installations. If you need different directory structure, refer to [README.md of ArgoCD Apps Component](https://github.com/kubermatic/kubermatic/blob/main/charts/gitops/kkp-argocd-apps/README.md) to understand how you can customize this, if needed.
+
+### ArgoCD Apps
+We will install ArgoCD on both the clusters and we will install following components on both clusters via ArgoCD. In non-GitOps scenario, some of these components are managed via kubermatic-installer and rest are left to be managed by KKP administrator in master/seed clusters by themselves. With ArgoCD, except for kubermatic-operator, everything else can be managed via ArgoCD. Choice remains with KKP Administrator to include which apps to be managed by ArgoCD.
+
+1. Core KKP components
+    1. DEX (in master)
+    1. ngix-ingress-controller
+    1. certManager
+1. Backup components
+    1. Velero
+1. Seed monitoring tools
+    1. Prometheus
+    1. AlertManager
+    1. Grafana
+    1. KubeStateMetrics
+    1. NodeExporter
+    1. BlackboxExporter
+    1. Identity aware proxy (IAP) for seed monitoring  components
+1. Logging components
+    1. Promtail
+    1. Loki
+1. S3 like object storage - minio
+1. User-cluster mla components
+    1. Minio and Minio Lifecycle Manager
+    1. Grafana
+    1. Consul
+    1. Cortex
+    1. LokiDistributed
```

Comment:
The fact that's it's the distributed form doesn't matter here, this can just be "Loki" again.